frontend/src/components/projectCreate/projectCreationMap.js

```diff
@@ -50,9 +51,10 @@ const BasemapMenu = ({ map }) => {
   );
 };
 
-const ProjectCreationMap = ({ mapObj, setMapObj, metadata, updateMetadata, step }) => {
+const ProjectCreationMap = ({ mapObj, setMapObj, metadata, updateMetadata, step, uploadFile }) => {
   const mapRef = React.createRef();
   const locale = useSelector((state) => state.preferences['locale']);
+  const { getRootProps, getInputProps } = useDropzone({ onDrop: uploadFile });
```

Comment:
@sumukhah you should use here the same dropzone config that I used on SetAoi. Your config doesn't allow us to draw on the map.